### ISSUE TITLE
[Request] Add fallback_resolver config and service information

### ISSUE THREAD

**Issue by iwan-wijaya** [bitbeans/SimpleDnsCrypt#268]
SimpleDnsCrypt version: 5.3 (latest)

I noticed that sometimes my dnscrypt-proxy service fails to start. The option is grayed out and usually also fails if started manually by clicking on it. This happens every couple days when I turn on my PC.
![screenshot_6](https://user-images.githubusercontent.com/11938557/37594179-6fc1f9f6-2baf-11e8-8483-f4dfceaa7538.png)

So I checked the logs: (logs are from my dnscrypt-proxy on a linux machine, but it should be the same)

```
[2018-03-19 10:03:29] [NOTICE] System DNS configuration not usable yet, exceptionally resolving [download.dnscrypt.info] using fallback resolver [9.9.9.9:53]
[2018-03-19 10:03:29] [CRITICAL] Unable use source [public-resolvers]: [dial udp 9.9.9.9:53: connect: network is unreachable]
[2018-03-19 10:03:29] [FATAL] No servers configured
```

Apparently at certain times dnscrypt-proxy service uses `fallback_resolver` (Default: Quad9/9.9.9.9:53) and fails. In my case, this is because in my country there is a government regulation that ISPs must only allow filtered DNS servers over port 53 to apply censorship.

Changing `fallback_resolver` in dnscrypt-proxy.toml to my ISP DNS fixes my issue.

So I think it will be a good idea to add `fallback_resolver` configuration to the GUI so we don't have to edit the dnscrypt-proxy.toml file.

It will be also great to have a notification in the GUI if the service fails to start and point out to the logs what may have gone wrong. Currently it is only grayed out without any information.

**Comment by raymondvillalobos** [comment]
can you please tell me what to edit in dnscrypt-proxy.toml?
I live in same country as you are.

**Comment by iwan-wijaya** [comment]
@raymondvillalobos I changed `fallback_resolver` to my ISP DNS.

**Comment by ardha225** [comment]
dude, can u show tutorial to fix that, sorry im newbie to the things like this. Problem with mine is error to start the service and fail to select any network cards.  Just dunno yet what happen on this new update. Tks

**Comment by nalyDylan** [comment]
@ardha225 

1. Go to: C:\Program Files\bitbeans\Simple DNSCrypt x64\dnscrypt-proxy
2. Open dnscrypt-proxy.toml with your favorite text editor (notepad works fine)
3. Change fallback_resolver to a different IP address, your router's IP should work if you know it's IP address

**Comment by undead098** [comment]
tried this fix. tried many dns address still can't start service.
is there any specific dns to try? i might miss something.

**Comment by jedisct1** [comment]
@undead098 If you are in Indonesia, this is a different problem.

**Comment by undead098** [comment]
@jedisct1 considering i got exact same error like on 1st post. maybe not entirely different problem.
i was able to fix "service cannot start" issue by reverting any customized DNS setting to default (including in router settings) and change `fallback-resolver` to my router IP.

**Comment by bitbeans** [comment]
fixed in 0.5.4
